### PR TITLE
Add tested Mac (M1) support

### PR DESCRIPTION
**Pull request by noise-puppeteer**

### CODE REVIEW

File: txt2vectorgfx.py [hunk 1]
```diff
@@ -99,15 +99,24 @@ def run(self, p, poFormat, poOpaque, poTight, poKeepPnm, poThreshold):
...
         # For Linux, run potrace from installed binary
-        if platform == "linux" or platform == "linux2":
+        if platform == "darwin":
+            try:
+                # check whether already in PATH 
+                checkPath = subprocess.Popen(["potrace","-v"])
+                checkPath.wait()
+                return "potrace"
+            except (Exception):
+                raise Exception("Cannot find installed Protrace on Mac. Please run `sudo apt install potrace`")
```
Comment (GeorgLegato) [inline txt2vectorgfx.py:109]: be Mac aware here as well please. 
if MAC, then "brew install potrce" ? I dont know exactly

Comment (noise-puppeteer) [reply txt2vectorgfx.py:109]: I'm sorry, the leftover code w('sudo') was an oversight. You are correct - it's [at Brew](https://formulae.brew.sh/formula/potrace):

        brew install potrace

The above installs generally to `/opt/homebrew/bin/potrace`, and is accessible generally from any path. Should I push another commit and make another pull request to you?

Comment (GeorgLegato) [reply txt2vectorgfx.py:109]: you can update into this PR , right?

Comment (GeorgLegato) [reply txt2vectorgfx.py:109]: just change your local file and push it

**Pull request merged**